components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.v1/src/main/resources/secretManager.yaml

```diff
@@ -0,0 +1,657 @@
+                $ref: '#/components/schemas/Error'
+        "500":
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    delete:
+      tags:
+        - Secret
+      summary: Delete an secret by name
+      description: |
+        This API provides the capability to delete a secret by
+        name.
+      operationId: deleteSecret
+      parameters:
+        - name: secret-type
+          in: path
+          description: name of the secret type
+          required: true
+          style: simple
+          explode: false
+          schema:
+            type: string
+        - name: name
+          in: path
+          description: name of the secret
+          required: true
+          style: simple
+          explode: false
+          schema:
+            type: string
+      responses:
+        "204":
+          description: No Content
+        "400":
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "401":
+          description: Unauthorized
+        "403":
+          description: Forbidden
+        "404":
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "405":
+          description: Method Not Allowed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "500":
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+components:
+  schemas:
+    Error:
+      type: object
+      properties:
+        code:
+          type: string
+          example: NSM-00000
+        message:
+          type: string
+          example: Some Error Message
+        description:
+          type: string
+          example: Some Error Description
+        traceId:
+          type: string
+          example: e0fbcfeb-3617-43c4-8dd0-7b7d38e13047
+    secretAddRequest:
+      required:
+        - name
+        - value
+      type: object
+      properties:
+        name:
+          type: string
+          example: choreo-riskScore
+        value:
+          type: string
+          example: bgtehaldhjdevch
+        description:
+          type: string
+          example: Some Error Description
+
+    SecretResponse:
+      required:
+        - secretId
+        - secretName
+        - lastModified
+        - created
+        - type
+      type: object
+      properties:
+        secretId:
+          type: string
+          example: 30103923-923c-485f-a8f9-606398
+        secretName:
+          type: string
+          example: choreo-riskScore
+        created:
+          type: string
+          example: 2021.07.22
```

Comment:
Does this format matches with what's actually being sent in the API response ?

Reply:
Addressed with commit [e65ca6f](https://github.com/wso2/identity-api-server/pull/315/commits/e65ca6f4f63741af54936deccf8f5f8c326fa8f0)